src/components/Breadcrumb/Breadcrumb.tsx

```diff
@@ -0,0 +1,77 @@
+/** @jsx jsx */
+import { jsx } from '@emotion/core';
+import React, { useMemo } from 'react';
+import { breadcrumbStyles, breadcrumbLinkStyles } from './Breadcrumb.style';
+import useTheme from 'hooks/useTheme';
+import BreadcrumbItem from './BreadcrumbItem/BreadcrumbItem';
+import isEmpty from 'lodash/isEmpty';
+import uniqueId from 'lodash/uniqueId';
+import { Link } from 'react-router-dom';
+import BreadcrumbCollapsed from './BreadcrumbCollapsed/BreadcrumbCollapsed';
+import { getLastDataItemProperty } from '../../utils/helpers';
+import { BreadcrumbItemData } from './types';
+
+export type Props = {
+  /** Defines the data for constructing the related breadcrumb items */
+  data: BreadcrumbItemData[];
+};
+
+const Breadcrumb: React.FC<Props> = props => {
+  const { children, data = [] } = props;
+  const theme = useTheme();
+  const passDataToRouterLink = ({ to, label }: BreadcrumbItemData) => (
+    <Link css={breadcrumbLinkStyles()(theme)} key={to} to={to}>
+      {label}
+    </Link>
+  );
+  const childrenCollection = React.Children.toArray(children);
+  const dataItems = isEmpty(data) ? childrenCollection : data.map(passDataToRouterLink);
+
+  const isLastItem = (dataItems: React.ReactNode[], itemIndex: number) =>
+    itemIndex === dataItems.length - 1;
+  const shouldCollapse = (item: React.ReactNode, itemIndex: number) =>
```

Comment:
please add a small description above this to explain the logic 
e.g Checks if items are between 1-3 and if should break the breadcrumb items

just for future reference in 3-4 months this would be helpful